R/string_operation.R

```diff
@@ -373,3 +373,15 @@ get_sentiment <- function(text){
   loadNamespace("sentimentr")
   sentimentr::sentiment_by(text)$ave_sentiment
 }
+
+#' Wrapper function for readr::parse_parse_character.
+#' @param text to parse
+#' @export
+parse_character <- function(text, ...){
+  # For non-character, use base as.character since readr::parse_character fails
```

Comment:
Please add comment about the history of this. We moved from readr 1.1.1 to 1.3.1 this time, and there taking character as input stopped working.